modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/PythonAbstractConnexionServerCodegen.java

```diff
@@ -708,6 +709,7 @@ public void setParameterExampleValue(CodegenParameter p) {
 
     public void setPackageName(String packageName) {
         this.packageName = packageName;
+        packagePath = packageName.replace('.', File.separatorChar);
```

Comment:
It seems a bit unnatural to do it in setPackagName.

What about adding a new function `toPackagePath()` to replce . with file separator without introducing another variable "packagePath"?